elements/rh-spinner/rh-spinner.ts

```diff
@@ -37,16 +43,19 @@ export class RhSpinner extends LitElement {
    @colorContextConsumer()
    @property({ reflect: true }) on?: ColorTheme;
 
+   /** Preset sizes for the spinner */
+  @property({ reflect: true }) size: SpinnerSize = 'lg';
+
 
-   render() {
-     return html`
```

Comment:
there were a bunch of fixable spacing lint errors in the files when I pulled them down. I'd be glad to help you set up your editor to automatically fix those kinds of errors

Reply:
Yes, that'd be great!